oar/core/advisory.py

```diff
@@ -749,3 +751,52 @@ def get_overall_grade(self):
         else:
             raise AdvisoryException(
                 f"error when accessing advisory containers - {self.errata_id}")
+
+    def check_kernel_tag(self):
+        # get advisories url
+        """
+        Get kernel tag from advisory build image.
+
+        Returns:
+            bool: True if kernel tagged with early-kernel-stop-ship.
+        """
+        build_url = "https://errata.devel.redhat.com/api/v1/erratum/" + str(self.errata_id) + "/builds"
+        logger.info(f"build url: {build_url}")
+        session = requests.Session()
+        session.hooks['response'] = lambda response, *args, **kwargs: response
+        session.auth = ('', '')
+        kerberos_auth = HTTPKerberosAuth(mutual_authentication=REQUIRED)
+        try:
+            response = session.get(build_url, auth=kerberos_auth)
+            if response.status_code == 401:
+                logger.error("401 Unauthorized: Authentication failed or missing.")
+            else:
+                try:
+                    rhos_nvr = jq.compile('.. | objects | to_entries[] | select(.key | startswith("rhcos-x86")) | .key').\
+                    input(text=response.text).all()
+                except jq.SyntaxError as e:
+                    logger.error(e)
+        except BaseException as e:
+            logger.error(e)
+        if len(rhos_nvr)==0:
```

Comment:
Could you please check code formatting? There are few more occurrences. 
```suggestion
        if len(rhos_nvr) == 0:
```